File: LIS.Api/Models/DenominationSearchResult.cs
namespace LIS.Api.Models
{
    public class DenominationSearchResult
    {
        public string? Insurance { get; set; }
        public int InsId { get; set; }
        public string? CostCenterName { get; set; }
        public int CostCenterId { get; set; }
        public int DenId { get; set; }
        public string? ActCode { get; set; } // Denomination.Code
        public string? ActName { get; set; } // SmallDescription
        public string? LabTest { get; set; }
        public decimal? CoefficientValue { get; set; }
        public decimal OutLL { get; set; }
        public decimal OutUsd { get; set; }
        public decimal PriceLL { get; set; }
        public decimal PriceUsd { get; set; }
        public bool? HasOperatingPhysician { get; set; }
    }
}

File: LIS.Web/src/app/models/denomination-search-result.ts
export interface DenominationSearchResult {
  insurance?: string | null;
  insId: number;
  costCenterName?: string | null;
  costCenterId: number;
  denId: number;
  actCode?: string | null;
  ActCode?: string | null;
  code?: string | null;
  actName?: string | null; // SmallDescription
  labTest?: string | null;
  coefficientValue?: number | null;
  outLL: number;
  outUsd: number;
  priceLL: number;
  priceUsd: number;
  hasOperatingPhysician?: boolean | null;
}

